model/resource/oci.yaml

```diff
@@ -0,0 +1,22 @@
+groups:
+  - id: oci
+    prefix: oci
+    type: resource
+    brief: >
+      An OCI instance.
+    attributes:
+      - id: manifest.digests
+        type: string
+        brief: >
+          The digest(s) of the OCI image manifest. For container images specifically it can be one or more
```

Comment:
Do we have more than one "digest"? If it is more, shouldn't this be an array then? The name/type is a bit contradicting right now.

Reply:
Good catch, thanks!

The intention is to be plural and hence it should be an array. Fixed the type with [9e3d8fb](https://github.com/open-telemetry/semantic-conventions/pull/159/commits/9e3d8fb4ada03898a25f03085a3495cf2d9535b8).

From container runtimes we can get more then one digests related to a specific running image. Hence we need to have it as an array to report multiple values.

From the repository perspective we can either report only the basic digest or if more digests are available (from additional image layers maybe) report all of them as well. @lmolkova would that make sense?

Reply:
Makes sense! Feel free to close this.

Reply:
sorry, I did not notice this  - the image has exactly one manifest (which can itself refer to a list of manifests)

Reply:
I think this is resolved now. Closing.